src/node_process_methods.cc

```diff
@@ -289,18 +274,37 @@ void GetActiveHandles(const FunctionCallbackInfo<Value>& args) {
       Array::New(env->isolate(), handle_v.data(), handle_v.size()));
 }
 
-void GetActiveHandlesInfo(const FunctionCallbackInfo<Value>& args) {
+static void GetActiveResourcesInfo(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
+  std::vector<Local<Value>> resources_info;
```

Comment:
I wonder if you can omit using `std::vector` for better performance in here.

Reply:
Any suggestions on how you think that can be done?